aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,213 @@
+
+[!code-xml[](target-aspnetcore/samples/single-tfm/netstandard2.0-tag-helpers-library.csproj)]
+
+### View components
+
+A project that includes [View components](xref:mvc/views/view-components) should use the `Microsoft.NET.Sdk` SDK. If targeting .NET Core 3.0, add a `<FrameworkReference>` element for the shared framework. For example:
+
+[!code-xml[](target-aspnetcore/samples/single-tfm/netcoreapp3.0-basic-library.csproj)]
+
+If targeting .NET Standard, add a package reference to [Microsoft.AspNetCore.Mvc.ViewFeatures](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.ViewFeatures). The `Microsoft.AspNetCore.Mvc.ViewFeatures` package moved into the shared framework and is therefore no longer published. For example:
+
+[!code-xml[](target-aspnetcore/samples/single-tfm/netstandard2.0-view-components-library.csproj)]
+
+## Support multiple ASP.NET Core versions
+
+Multi-targeting is required to author a library that supports multiple variants of ASP.NET Core. Consider a scenario in which a Tag Helpers library must support the following ASP.NET Core variants:
+
+* ASP.NET Core 2.1 targeting .NET Framework 4.6.1
+* ASP.NET Core 2.x targeting .NET Core 2.x
+* ASP.NET Core 3.0 targeting .NET Core 3.0
+
+The following project file supports these variants via the `TargetFrameworks` property:
+
+[!code-xml[](target-aspnetcore/samples/multi-tfm/recommended-tag-helpers-library.csproj)]
+
+With the preceding project file:
+
+* The `Markdig` package is added for all consumers.
+* A reference to [Microsoft.AspNetCore.Mvc.Razor](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor) is added for consumers targeting .NET Framework 4.6.1 or .NET Core 2.x. Version 2.1.0 of the package works with ASP.NET Core 2.2 because of backwards compatibility.
+* The shared framework is referenced for consumers targeting .NET Core 3.0. The `Microsoft.AspNetCore.Mvc.Razor` package is included in the shared framework.
+
+Alternatively, .NET Standard 2.0 could be targeted instead of targeting both .NET Core 2.1 and .NET Framework 4.6.1:
+
+[!code-xml[](target-aspnetcore/samples/multi-tfm/alternative-tag-helpers-library.csproj?highlight=4)]
+
+With the preceding project file, .NET Core 2.x and .NET Framework 4.6.1 projects are supported because both target frameworks implement .NET Standard 2.0. Since this library only contains Tag Helpers, it's more efficient to target the specific platforms on which ASP.NET Core runs: .NET Core and .NET Framework. Tag Helpers can't be used by other .NET Standard 2.0-compliant target frameworks such as Unity, UWP, and Xamarin. If your library needs to call platform-specific APIs, target specific .NET implementations instead of .NET Standard. For more information, see [Multi-targeting](/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting).
```

Comment:
Replace "efficient" with "straightforward".

Also, we generally recommend that class libraries directly target .NET Framework to support it, rather than .NET Standard, due to the issues of using .NET Standard targeted packages on .NET Framework (this is covered in the general library guidance too I believe).